test/metabase/pulse/render/js_svg_test.clj

```diff
@@ -285,28 +286,27 @@
         rows2       [[#t "2000-03-01T00:00:00Z" 3]
                      [#t "2002-03-01T00:00:00Z" 4]]
         ;; this one needs more stuff because of stricter ts types
-        series-seqs [[{:name          "bob"
-                       :color         "#cccccc"
+        series-seqs [[{:cardName      "bob"
                        :type          "area"
                        :data          rows1
                        :yAxisPosition "left"
-                       :column        {:name "count"}}
-                      {:name          "bob2"
-                       :color         "#cccccc"
+                       :column        {:name "count" :display_name "Count"}}
```

Comment:
for series with multiple metrics `display_name` is used to render legends.
https://github.com/metabase/metabase/blob/9b9c6d17d1f919bb88608b55be0c33b46d635bee/frontend/src/metabase/static-viz/components/LineAreaBarChart/utils/series.tsx#L86

Without this, FE will fail to render because `name` will be null, and it won't be able to to calculate its width.